src/components/main-view/main-view.jsx

```diff
@@ -20,11 +21,20 @@ class MainView extends React.Component {
     super();
     this.state = {
       movies: [],
-      selectedMovie: null,
```

Comment:
hey, please dont remove this section as it will break the app or something is depending on it